autoload/neomake/makers/ft/scala.vim

```diff
@@ -1,11 +1,23 @@
 " vim: ts=4 sw=4 et
 function! neomake#makers#ft#scala#EnabledMakers()
-    let makers = ['scalac']
-    if neomake#utils#Exists('scalastyle')
+    let makers = get(g:, 'neomake_scala_fast', 0) ? ['fsc'] : ['scalac']
```

Comment:
I believe when you install scala you will have both on your path. So you'd have to `let g:neomake_scala_enabled_makers = ['fsc']` the default should be scalac.